service/src/main/java/de/adorsys/psd2/sandbox/portal/app/ConfigService.java

```diff
@@ -0,0 +1,45 @@
+package de.adorsys.psd2.sandbox.portal.app;
+
+import de.adorsys.psd2.sandbox.portal.app.domain.ContentUrls;
+import de.adorsys.psd2.sandbox.portal.app.domain.UiConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ConfigService {
+
+  private final Environment environment;
+
+  @Autowired
+  public ConfigService(Environment environment) {
+    this.environment = environment;
+  }
+
+  /**
+   * Returns the Config for the portal.
+   *
+   * @return UiConfig
+   */
+  UiConfig getUiConfig() {
```

Comment:
Could make sense to bind the properties directly to `UiConfig` to be able to validate them. But good enough for now.

Reply:
Done